### PR TITLE
adds pill packs

### PR DESCRIPTION
**Pull request by Andromeda-K22**
🆑 Adds packs of pills, to be added to first aid kits / pouches, maybe? Use in hand to pop a pill out, examine shows the pills they contain. 

To be expanded to chemistry, for now, DNM.

🆑

### CODE REVIEW

File: baystation12.dme [hunk 1]
```diff
@@ -1106,7 +1106,7 @@
 #include "code\game\objects\items\weapons\storage\storage_boh.dm"
 #include "code\game\objects\items\weapons\storage\toolbox.dm"
 #include "code\game\objects\items\weapons\storage\uplink_kits.dm"
-#include "code\game\objects\items\weapons\storage\wall_mirror.dm"
```
Comment (Andromeda-K22) [inline baystation12.dme]: `#include "code\game\objects\items\weapons\storage\wall_`

**Comment by Roland410** [comment]
Soooo, pill bottles you mean? Seems like a redundant item that does the same thing, but not reusable?

**Comment by Sparrowbird611** [comment]
What roland said. What's the point of this, exactly? Chemists already have like two boxes of pill bottles, and pill requests are rare in and of themselves. Seems like needless bloat.

**Comment by Andromeda-K22** [comment]
These are slightly easier to use than pill bottles, also, this was requested by RenGusta and is a precursor to another item which is not intended to contain pill bottles.

This is to be used in situations where giving an entire bottle of bills (often +200 units of a medicine!) on prespawn would be overpowered.

**Comment by Roland410** [comment]
Or you could just subtype the pill bottle/preassign the amount of pills it has. This is basically code bloat that can be solved with existing items way easier.

**Comment by Andromeda-K22** [comment]
I'm unwilling to consider 87 lines of code 'code bloat'.

**Comment by MistakeNot4892** [comment]
Possible alternative implementation in the PR linked above.

I quite like these, the little pop is a lot more satisfying than just getting some loose pills in a pouch.

**Comment by BurgerLUA** [comment]
I'm waiting for the label to be removed before I can start reviewing.

**Pull request closed without merging**